hack/e2e.sh

```diff
@@ -102,5 +116,24 @@ while [ $ATTEMPTS -ge 0 ]; do
   fi
 done
 
+# Wait for cincinnati metrics to be recorded
+# Find out the token Prometheus uses from its serviceaccount secrets
+# and use it to query for GB build info
+PROM_ENDPOINT=$(oc -n openshift-monitoring get route thanos-querier -o jsonpath="{.spec.host}")
+PROM_TOKEN=$(oc -n openshift-monitoring get secret \
+  $(oc -n openshift-monitoring get serviceaccount prometheus-k8s \
+    -o jsonpath='{range .secrets[*]}{.name}{"\n"}{end}' | grep prometheus-k8s-token) \
+  -o jsonpath='{.data.token}' \
+  -o go-template='{{.data.token | base64decode}}')
+
+ATTEMPTS=10
+DELAY=30
+while [ $ATTEMPTS -ge 0 ]; do
```

Comment:
possibly simpler than the `while` would be:

```sh
for i in $(seq 10); do
```

which would let you drop the later `ATTEMPTS=$((ATTEMPTS-1))`.